packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -434,44 +439,72 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
     }, [tileStyle]);
 
     /*
-     * React on changes in the primary color prop.
+     * React on changes in the primary color prop. If not defined in the URL, check if it is defined in app config. Otherwise set to default color.
```

Comment:
```suggestion
     * React on changes in the primary color prop. If not defined, check if it is defined in app config. Otherwise set to default color.
```